static/templates/actors/parts/actor-spellbook.hbs

```diff
@@ -1,11 +1,18 @@
 <div class="inventory-filters spellbook-filters">
+    {{#if isNPC}}
+    <div class="form-group" data-tooltip='<strong>{{localize "SFRPG.NPCSheet.Header.BaseSpellDC"}}</strong><br>"@attributes.baseSpellDC.value"<br><br>{{localize "SFRPG.NPCSheet.Header.BaseSpellDCTooltip"}}'>
+        <label>{{localize "SFRPG.NPCSheet.Header.BaseSpellDC"}}</label>
```

Comment:
<label><h3 class="attribute-name box-title">{{localize "SFRPG.NPCSheet.Header.BaseSpellDC"}}</h4></label>
        <input type="number" name="system.attributes.baseSpellDC.base" value="{{system.attributes.baseSpellDC.base}}" placeholder=0/>

Reply:
Might as well change the input type to number and looks weird to have the huge input box.